CHANGES.rst

```diff
@@ -22,14 +22,29 @@ Bug fixes:
 - *add item here*
 
 
+
 5.0.9 (2017-08-05)
 ------------------
 
 New features:
 
+- Add js-shortcuts package for keyboard shortcuts
+  [b4oshany]
+
+- Update package version to 5021 from 5020.
+  [b4oshany]
+
+- Create an upgrade step to 5020 from 5019.
+  [b4oshany]
```

Comment:
I believe the changelog changes should match the ones in https://github.com/plone/Products.CMFPlone/pull/2121